### PR TITLE
Update roomba.markdown

### PR DESCRIPTION
**Pull request by ghost**
Following the manual of this integration, it is too hard for unexperienced users to get the BLID and Password credentials with the current links on the bottom of the page: https://www.home-assistant.io/integrations/roomba/

I'm not an expert but I'd like to contribute with this solution to make it more easyer to get the BLID and Password. If it's not appriciated, or not according to the rules, then never mind. If this looks good we'll then publish! :-) Thanks and have a great day!
The only change is the extra text on the bottom. The rest is forked as-it-was.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by ghost** [comment]
@pschmitt can you review?

**Comment by frenck** [comment]
Would you please reframe from pinging people for review @MrUserY? Thanks 👍

**Comment by pschmitt** [comment]
@MrUserY there's a user currently working on automating the whole process via config flow (see [here](https://github.com/pschmitt/roombapy/issues/53)). Once this is ready (it's not yet) the manual retrieval of the password will no longer be necessary.

Other than that I do not have access to any window machine and cannot possibly review this PR.

File: source/_integrations/roomba.markdown [hunk 1]
```diff
@@ -107,3 +107,35 @@ docker exec -it CONTAINER_NAME_OR_ID python -c 'import roomba.entry_points; room
...
 </div>
+
+
+### Alternative option to retrieve Roomba Password and BLID using a regular Windows PC
+
+1. Downloading files
+Create a temporary folder on your desktop for extra convenience (you can name the folder whatever you want) and put these two files inside that folder:
+
+- Python installation package
+It's recommend getting the latest “executable installer”. You can find this in the ‘Download’ section of the official Python website https://www.python.org
```
Comment (frenck) [inline source/_integrations/roomba.markdown:118]: Do not use bare links.

File: source/_integrations/roomba.markdown [hunk 2]
```diff
@@ -107,3 +107,35 @@ docker exec -it CONTAINER_NAME_OR_ID python -c 'import roomba.entry_points; room
 The command to retrieve the credentials does not need any additional software to be installed because it uses the built-in [roombapy](https://github.com/pschmitt/roombapy) package and [password](https://github.com/pschmitt/roombapy/blob/1.6.1/roomba/entry_points.py#L20) funciton deployed with Home Assistant.
 
 </div>
+
+
+### Alternative option to retrieve Roomba Password and BLID using a regular Windows PC
+
+1. Downloading files
+Create a temporary folder on your desktop for extra convenience (you can name the folder whatever you want) and put these two files inside that folder:
```
Comment (frenck) [inline source/_integrations/roomba.markdown:115]: this is not indented thus not part of the list?

File: source/_integrations/roomba.markdown [hunk 3]
```diff
@@ -107,3 +107,35 @@ docker exec -it CONTAINER_NAME_OR_ID python -c 'import roomba.entry_points; room
...
+- Python installation package
+It's recommend getting the latest “executable installer”. You can find this in the ‘Download’ section of the official Python website https://www.python.org
+
+- Roomba Python Library
+This is a zip-package of files used to establish integration with your Roomba. You can find it at https://github.com/NickWaterton/Roomba980-Python. Look for a green button named “Code”, Click it and select “Download zip”.
+Now you should have two files (one exe file and one zip file) in a folder on your desktop.
+
+2. Install Python
+
+- a.	Start Python installation by running the exe-file. Select “Customize installation”. Click “Next”. On the “Advanced options” page be sure to check “Add Python to environment variables”, and edit “Customize install location” to `c:\Python`, then click “Install”. (Please keep the Python installation file in your desktop folder for now).
```
Comment (frenck) [inline source/_integrations/roomba.markdown:126]: Please don't use tabs.

File: source/_integrations/roomba.markdown [hunk 4]
```diff
@@ -107,3 +107,35 @@ docker exec -it CONTAINER_NAME_OR_ID python -c 'import roomba.entry_points; room
...
+### Alternative option to retrieve Roomba Password and BLID using a regular Windows PC
+
+1. Downloading files
+Create a temporary folder on your desktop for extra convenience (you can name the folder whatever you want) and put these two files inside that folder:
+
+- Python installation package
+It's recommend getting the latest “executable installer”. You can find this in the ‘Download’ section of the official Python website https://www.python.org
+
+- Roomba Python Library
+This is a zip-package of files used to establish integration with your Roomba. You can find it at https://github.com/NickWaterton/Roomba980-Python. Look for a green button named “Code”, Click it and select “Download zip”.
```
Comment (frenck) [inline source/_integrations/roomba.markdown:121]: Bare links should not be used.

File: source/_integrations/roomba.markdown [hunk 5]
```diff
@@ -107,3 +107,35 @@ docker exec -it CONTAINER_NAME_OR_ID python -c 'import roomba.entry_points; room
...
+Now you should have two files (one exe file and one zip file) in a folder on your desktop.
+
+2. Install Python
+
+- a.	Start Python installation by running the exe-file. Select “Customize installation”. Click “Next”. On the “Advanced options” page be sure to check “Add Python to environment variables”, and edit “Customize install location” to `c:\Python`, then click “Install”. (Please keep the Python installation file in your desktop folder for now).
+
+- b.	Open the Roomba zip-file and locate the “roomba” folder inside. Copy this folder into your freshly installed Python folder (c:\Python).
+
+- c.	Use your favourite method of opening a DOS prompt (example: click the magnifying glass besides the Start-menu icon and search for “cmd” or open a PowerShell window). Then use the DOS prompt to navigate to your C:\Python folder using the `cd \python` command.
+Once inside the Python folder, type `pip install six` and press ENTER. Wait for the ”six” installation to finish. After a few seconds you should be back in the DOS prompt.
```
Comment (frenck) [inline source/_integrations/roomba.markdown:131]: That will install six globally. Shouldn't that be avoided?

File: source/_integrations/roomba.markdown [hunk 6]
```diff
@@ -107,3 +107,35 @@ docker exec -it CONTAINER_NAME_OR_ID python -c 'import roomba.entry_points; room
...
+
+3. Run the script
+Type `python .\roomba\getpassword.py`, hit ENTER and wait a few seconds. Now your Roomba should be detected by the script. From here on you should follow on-screen directions (something about making sure your Roomba is turned on and parked in the base, and then holding the “Home” button for a few seconds to hear a bleep. This should cause the wifi indicator on your Romba to flash - then hit ENTER to continue and wait a few seconds more).
+
+- Your Roomba’s BLID and password now appears in the bottom of the DOS window! Copy this info to a txt-file, grab a screenshot or whatever method you like best. In case of no or bad response: Make shure no apps are connected to Roomba. Or try holding down the the standby button on the Roomba until it reboots.
+
+- From here on you can follow the instructions found on the Home Asistant website on how to add your Roomba to your “configuration.yaml” file (https://www.home-assistant.io/components/vacuum.roomba/). Then you’re ready to reboot your Home Assistant and enjoy Roomba integration.
```
Comment (frenck) [inline source/_integrations/roomba.markdown:138]: - bare link
- code fence filenames

File: source/_integrations/roomba.markdown [hunk 7]
```diff
@@ -107,3 +107,35 @@ docker exec -it CONTAINER_NAME_OR_ID python -c 'import roomba.entry_points; room
...
+
+4. Cleanup
+Start the Python installation by running the exe-file in your temporary desktop folder, select “Uninstall” and wait for uninstaller to finish. Now you can delete the ”C:\Python” folder, and also delete the temporary folder from your desktop.
```
Comment (frenck) [inline source/_integrations/roomba.markdown:141]: Please, code-fence paths

**Review by frenck — CHANGES_REQUESTED** [review]

**Comment by frenck** [comment]
Oh user is a ghost now. Closing.

**Pull request closed without merging**